Add Kujira POD (#128)
cosmos/awesome-cosmos: README.md

* [List on Mintscan](https://www.mintscan.io/cosmos/validators)
* [List on ATOMScan](https://atomscan.com/validators)
* [List on BigDipper](https://cosmos.bigdipper.live/validators)

> * [List on Kujira POD](https://pod.kujira.app/cosmoshub-4) 

Reviewer: ```suggestion
* [List on Kujira POD](https://pod.kujira.app/cosmoshub-4)

```

* [List on BigDipper](https://cosmos.bigdipper.live/validators)

> 

Reviewer: ```suggestion
* [List on Kujira POD](https://pod.kujira.app/cosmoshub-4)

```

DYOR when choosing a validator. Consider delegating your tokens to validators outside of the top 20 to increase the decentralization of the network.

> Check validator voting power on [Kujira POD](https://pod.kujira.app/cosmoshub-4).

Reviewer: We shouldn't promote a specific service. The services mentioned above can do the same thing. I think it's best to just add it to the list.

> Check validator voting power on [Kujira POD](https://pod.kujira.app/cosmoshub-4).

Reviewer: ```suggestion
```

> This is also view as a good pratice to avoid 0% commision validators and exchanger validators.

Reviewer: ```suggestion
This is also a good pratice to avoid 0% commission validators and exchange validators.
```
